script/entrypoint

```diff
@@ -19,7 +19,8 @@ fi
 
 if [ "$#" = 0 ]; then
     # Default to 20 workers because most of the tests are io-bound
-    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle --numprocesses 20
+    # Ignore running routine and SQL tests since they require GCP access
+    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle --numprocesses 20 -m "not (routine or sql or integration or java)"
```

Comment:
We specified custom markers for [routine](https://github.com/mozilla/bigquery-etl/blob/374325adc7abf87ea179eacd2fde5ecc6c1f6779/bigquery_etl/pytest_plugin/routine.py#L38) and [SQL tests](https://github.com/mozilla/bigquery-etl/blob/374325adc7abf87ea179eacd2fde5ecc6c1f6779/bigquery_etl/pytest_plugin/sql.py#L31). I think it should be fine ignoring them here. The only problem I see is that these tests will also get ignored when running tests locally.

Reply:
This does seem like it will require at least an update to the testing docs: https://github.com/mozilla/bigquery-etl/blob/main/tests/README.md#how-to-run-tests

What's the invocation we would need to run a udf test now?